tests/behat/features/asp/ondemand-unauth.feature

```diff
@@ -64,21 +64,22 @@ Feature: This Behat test checks the OnDemand page
     Then I should see "Learn more about OnDemand"
     Then I should see an image with alt text "Laptop team"
     Then I should see "ACCESS OnDemand helps computational researchers and students efficiently"
-    #TODO Need to test link to Learn More. There are two Learn mre buttons
+    # test Learn More, including link
     Then I should see "Learn More"
+    When I go to "http://openondemand.org/"
```

Comment:
Can we use the a[href=] here?

Reply:
are you saying you like 
  When I click the "a[href='http://openondemand.org/']" element
more than 
  When I go to "http://openondemand.org/"
?

Reply:
Yes. When I go to ondemand is not testing our website at all.

Reply:
ahh, of course, very excellent point.  

There are 242 instances of "When I go to"...  I think these should all get reviewed.  Some of them are more in the spirit of "When I am on ...", and can be switched to that.  

I think I should make a pass at updating all of those, which I would do a new branch.   Agreed?

Reply:
I think most of those are going back to the original page that was being testing after checking a link? Probably doesn't matter if it is phrased "when i go to" or "when i am on", but maybe I'm missing something?

Reply:
from the web:
```The difference between "when I go to" and "when I am on" in Behat is that "when I go to" is used to describe the action of navigating to a specific page, while "when I am on" is used to describe the state of being on a specific page.
For example, the following step would use "when I go to":
When I go to the homepage 
This step would tell Behat to navigate to the homepage of the application.
The following step would use "when I am on":
When I am on the homepage 
This step would tell Behat to check if the current page is the homepage.
The choice of which keyword to use depends on the context of the step. If the step is describing an action, then "when I go to" should be used. If the step is describing a state, then "when I am on" should be used.```

Reply:
Ok, I looked it over, and yes, most are just going back to the main testing page.  I agree, either is fine.  Not worth changing.